.github/workflows/publish-latest.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     steps:
       - name: Checkout repo
         uses: actions/checkout@v2
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
```

Comment:
This is interesting.  I'm trying to understand what behavior prompted this.

My understand was that the release PR (https://github.com/aws-amplify/amplify-ui/pull/596) only needs `HEAD` since `.changesets` has all the files there and it's able to calculate it without a huge history.

Is there a comment or reference here? It seems like an easy thing to miss.

Also, is changeset re-calculating versions from the entire history? Or just from `HEAD`? 🤔

Reply:
To be perfectly honest, I just took this from https://github.com/changesets/action#with-publishing. And this actions haven't had problem so far with generating changelog on #596, so I'm not sure this is necessary.

Reply:
Let's keep it since it's consistent with https://github.com/changesets/action#with-publishing.